lib/charms/mysql/v0/mysql.py

```diff
@@ -67,7 +67,6 @@ def wait_until_mysql_connection(self) -> None:
 
 import json
 import logging
-import pathlib
```

Comment:
The libpatch for this lib also needs to be updated

Reply:
fixed in https://github.com/canonical/mysql-operator/pull/204/commits/9a9f98955366f475ffbf6321a89b6a61260d9a50